github/actions/multi_client.go

```diff
@@ -168,3 +169,16 @@ func (m *multiClient) GetClientFromSecret(ctx context.Context, githubConfigURL,
 	auth.AppCreds = &GitHubAppAuth{AppID: parsedAppID, AppInstallationID: parsedAppInstallationID, AppPrivateKey: appPrivateKey}
 	return m.GetClientFor(ctx, githubConfigURL, auth, namespace)
 }
+
+func RootCAsFromConfigMap(configMapData map[string][]byte) (*x509.CertPool, error) {
+	caCertPool := x509.NewCertPool()
```

Comment:
should we default to `x509.SystemCertPool()` so we can get all default certs on the machine?